packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/avatar.tests.js

```diff
@@ -40,25 +40,27 @@ QUnit.module('ChatAvatar', moduleConfig, () => {
         QUnit.test('should be rendered with correct initials according passed name option value', function(assert) {
             this.reinit({ name: 'User name' });
 
-            assert.strictEqual(this.$element.text(), 'U');
+            assert.strictEqual(this.$element.text(), 'UN');
         });
 
         QUnit.test('name option should be updatable at runtime', function(assert) {
             this.instance.option('name', 'New Value');
 
-            assert.strictEqual(this.$element.text(), 'N');
+            assert.strictEqual(this.$element.text(), 'NV');
         });
 
         [
+            { name: 'onewordname', expectedInitials: 'O' },
+            { name: 'Three word name', expectedInitials: 'TW' },
             { name: 888, expectedInitials: '8' },
             { name: undefined, expectedInitials: '' },
             { name: null, expectedInitials: '' },
-            // TODO: consider scenarios
-            // { name: ' New Name', expectedInitials: 'N' },
-            // { name: NaN, expectedInitials: '' },
-            // { name: Infinity, expectedInitials: '' },
-            // { name: -Infinity, expectedInitials: '' },
-            // { name: { firstName: 'name' }, expectedInitials: '' }
+            { name: ' New Value', expectedInitials: 'NV' },
+            { name: '   New      Value.   ', expectedInitials: 'NV' },
+            { name: NaN, expectedInitials: '' },
+            { name: Infinity, expectedInitials: '' },
+            { name: -Infinity, expectedInitials: '' },
+            { name: { firstName: 'name' }, expectedInitials: '' }
```

Comment:
[testing coverage]
As far as i see there is no visual test at the moment for 2-letter Avatar. Suggest u adding it

Reply:
[7817343832dae488956ab0205032caab174d0ee2], [48b236474f079d7aad61cb0d15d26a3a02c04d84]